assets/js/scenes/department/ProgramEconSmallMultiples/programme-econ-small-muls.html

```diff
@@ -18,7 +18,7 @@ <h3 class="Title Title--section">Programme spending under economic classificatio
             <div class="ProgramEconSmallMultiplesSection-share u-marginTop10 u-marginBottom15">
                 <div data-component="Share" data-anchor="econ-classification" data-color="{{ color }}"></div>
             </div>
-
+            {% if subprog_econ4_bars_data.get_dataset != None %}
```

Comment:
I don't htink we need changes in this section - It's also the budget vs actuals dataset

Reply:
changed done.